requirements.txt

```diff
@@ -0,0 +1,9 @@
+tensorflow>=2.4
```

Comment:
`requirements.txt` is not necessary because we define dependencies in `pyproject.toml`.

https://github.com/faustomorales/vit-keras/blob/a37c49e07ca9504a00abacd82762fc5b020a97c1/pyproject.toml#L28-L31

The dependencies list excludes `opencv-python` and `tensorflow` because those dependencies come in *many* different flavors (e.g., `opencv-python` vs `opencv-python-headless` vs `opencv-contrib-python`) and it's best to leave it to users to decide which ones they want separately.

To install this repository from GitHub, there is no need to do `git clone` / `pip install requirements.txt` / `sys.path.insert`. Instead, you can use a command like the following which will let `pip` take care of cloning, switching branches, installing dependencies, and installing the package so you can import it.

```
pip install git+https://github.com/awsaf49/vit-keras.git@reg-custom-layer#egg=vit_keras
```

Alternatively, if you want to do the `git clone` yourself, you can do that and then run `pip install path/to/repo/dir` to install. In either case, `pip` will install dependencies from `pyproject.toml`.